docs/docs/validators/partial-set-security-for-validators.md

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 6
+---
+
+# Partial Set Security
+
+[Partial Set Security](../features/partial-set-security.md) allows consumer chains to join as Opt-In or Top N.
+Here, we show how a validator can opt in, opt out, or set a custom commission rate on a consumer chain, as well
+as useful queries that a validator can use to figure out which chains it has to validate, etc.
+
+## Messages
+### How to opt in to a consumer chain?
+
+:::warning
+A validator is automatically opted in to a Top N chain if the validator belongs to the top N% of the validators on the provider chain.
```

Comment:
We should have another warning along the lines of:
Validators that move out of the top N% of the validators on the provider chain are not opted out automatically, and must opt out before they can stop their nodes.

Reply:
Absolutely, thanks for pointing out